common/models/role.js

```diff
@@ -31,19 +28,19 @@ module.exports = function(Role) {
     return new Date();
   };
 
+  Role.getRelatedModels = function() {
```

Comment:
same concern as `RoleMapping` version of this... I'm guessing there is something preventing from doing this is a more standard way... Can you elaborate?